core/src/main/java/org/sql2o/converters/LocalDateConverter.java

```diff
@@ -0,0 +1,39 @@
+package org.sql2o.converters;
+
+import java.sql.Date;
+import java.time.LocalDate;
+
+/**
+ * Used by sql2o to convert a value from the database into a {@link java.time.LocalDate}.
+ *
+ * @since 13/2/2024
+ * @author Agit Rubar Demir | @agitrubard
+ * @version 1.8.0
+ */
+public class LocalDateConverter implements Converter<LocalDate> {
+
+    @Override
+    public LocalDate convert(Object dateObject) throws ConverterException {
+
+        if (dateObject == null) {
+            return null;
+        }
+
+        try {
+            return LocalDate.parse(dateObject.toString().split(" ")[0]);
```

Comment:
Converting to a string before converting to a `LocalDate` seems like a sub-optimal conversion. Most of the time, a Jdbc driver will return a `java.sql.Date` which have a `toLocalDate` method.

Reply:
In our previous applications, the solutions I applied indicated that different approaches could be implemented in different databases. Currently, I believe this is the most optimal solution. However, if you have tried and reached a definite solution that you have confidence in, we can certainly use that.